documentation/docs/proptest/extra_arbs.md

```diff
@@ -51,11 +55,12 @@ To use, add `io.kotest.extensions:kotest-property-arbs:version` to your build.
 | Arb.cluedoWeapons()        | Clue/Cluedo weapons, eg `Lead piping`                                                                               |
 | Arb.cluedoLocations()      | Clue/Cluedo locations, eg `Ballroom`                                                                                |
 | Arb.cluedoAccusation()     | Clue/Cluedo accusations, eg, `Mrs White / Billiards Room / Rope`                                                    |
-| Arb.monopolyProperty()     | Random (US version) monopoly property with rent, price and color                                                    |
 | Arb.chessPiece()           | Chess piece with points                                                                                             |
 | Arb.chessSquare()          | Chesss square with file A-H and rank 1-8                                                                            |
 | Arb.chessMove()            | Chess move from square to square with captured piece if any. No validation is performed to check the move is legal. |
 |                            |                                                                                                                     |
 | Arb.transactions()         | Transactions with a card number, card type, amount and transaction type                                             |
 |                            |                                                                                                                     |
 | Arb.cars()                 | Random car manufacturers                                                                                            |
+| 	                         |                                                                                                                     |
+| Arb.googleTaxonomy         | Structured taxonomy of random things                                                                                |
```

Comment:
do we need `()` parenthesis after function name?

Reply:
fixed